int16[] subject
float64[] range
float64[] bearing